### PR TITLE
Update comments for Chinese document.

### PR DESCRIPTION
**Pull request by natural-law**
@samuele3hu 
Please review this PR.

### CODE REVIEW

File: cocos/base/CCUserDefault.h [hunk 1]
```diff
@@ -36,173 +36,262 @@ THE SOFTWARE.
...
-    /**
+    /**@~english
      * Get bool value by key, if the key doesn't exist, will return false.
      * You can set the default value, or it is false.
-     * @param key The key to get value.
-     * @return Bool value by `key`.
+     * @~chinese 
+     * 获取指定 key 的布尔型值，如果 key 不存在，则返回 false。
+     * @param key @~english The key to get value.
+     * @~chinese 需要获取的关键字。
```
Comment (samuele3hu) [inline cocos/base/CCUserDefault.h]: 关键字可以不用翻译吧？

File: cocos/base/CCUserDefault.h [hunk 2]
```diff
@@ -36,173 +36,262 @@ THE SOFTWARE.
...
+     * @param key @~english The key to set.
+     * @~chinese 需要设置的关键字。
+     * @param value @~english A bool value to set to the key.
+     * @~chinese 需要设置的值。
      * @js NA
      */
     virtual void setBoolForKey(const char* key, bool value);
-    /**
+    /**@~english
      * Set integer value by key.
```
Comment (samuele3hu) [inline cocos/base/CCUserDefault.h]: by -> for?

File: cocos/base/CCUserDefault.h [hunk 3]
```diff
@@ -212,15 +301,23 @@ class CC_DLL UserDefault
...
+     * and use this function to set delegate, then UserDefault will invoke delegate's implementation.
+     * For example, your store native data base or other format store.
+     *
+     * If you don't want to system default implementation after setting delegate, you can just pass nullptr
+     * to this function.
+     *
+     * @~chinese 
+     * 你可以通过继承 UserDefault 来实现自定义的数据管理类，比如：UserDefaultAndroid,
+     * 使用这个函数来设置代理对象，然后 UserDefault 将调用代理对象的实现。
+     * 例如：使用本地化的数据库或者数据格式来存储与读取数据。
```
Comment (samuele3hu) [inline cocos/base/CCUserDefault.h:315]: 英文注释要修改一下？

File: cocos/base/CCUserDefault.h [hunk 4]
```diff
@@ -212,15 +301,23 @@ class CC_DLL UserDefault
...
+     * 
+     * 如果在设置代理对象之后，想要不再使用代理对象。可以通过调用 `setDelegate(nullptr)` 来实现。
```
Comment (samuele3hu) [inline cocos/base/CCUserDefault.h:317]: don't want to system default implementation ->不再使用代理对象?

File: cocos/base/CCUserDefault.h [hunk 5]
```diff
@@ -233,12 +330,15 @@ class CC_DLL UserDefault
      * @js NA
      */
     CC_DEPRECATED_ATTRIBUTE static void purgeSharedUserDefault();
-    /** All supported platforms other iOS & Android use xml file to save values. This function is return the file path of the xml path.
+    /** @~english All supported platforms other iOS & Android use xml file to save values. This function is return the file path of the xml path.
```
Comment (samuele3hu) [inline cocos/base/CCUserDefault.h]: other ->except?

File: cocos/base/CCUserDefault.h [hunk 6]
```diff
@@ -233,12 +330,15 @@ class CC_DLL UserDefault
...
+     * @~chinese 除了 iOS 和 Android 之外的所有支持的平台都使用 xml 文件保存数据。这个函数返回 xml 文件的路径。
      * @js NA
      */
     static const std::string& getXMLFilePath();
-    /** All supported platforms other iOS & Android and CC_PLATFORM_WINRT use xml file to save values. This function checks whether the xml file exists or not.
-     * @return True if the xml file exists, flase if not.
+    /** @~english All supported platforms other iOS & Android and CC_PLATFORM_WINRT use xml file to save values. This function checks whether the xml file exists or not.
```
Comment (samuele3hu) [inline cocos/base/CCUserDefault.h]: other->except?

File: cocos/base/CCUserDefault.h (hunk 3 above)
Comment (samuele3hu) [inline cocos/base/CCUserDefault.h:315]: 无问题。

**Comment by natural-law** [comment]
@samuele3hu 
I have solved the problems.

**Comment by samuele3hu** [comment]
@andyque, it is ok.

**Pull request merged**